src/browser/provider/built-in/base.js

```diff
@@ -0,0 +1,75 @@
+import browserNatives from 'testcafe-browser-natives';
+import OS from 'os-family';
+import WARNING_MESSAGE from '../../../warnings/message';
+
+
+/*eslint-disable no-undef*/
+function getDataFromBrowser () {
+    return { width: window.innerWidth, height: window.innerHeight, title: document.title };
+}
+/*eslint-disable no-undef*/
+
+const initScript = getDataFromBrowser.toString();
```

Comment:
Why not inline initialization? Variable name doesn't follow const naming notation